aayushi/src/components/Label.jsx

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable react/prop-types */
+
+export const Label = ({ type, className }) => {
+  return (
+    <div className="mb-2">
+      <label className={`text-sm mb-2 ${className}`}>{type} </label>
+      <span className="ml-1 text-red-500">*</span>
```

Comment:
don't use red * here only as it should be placed only on conditional basis if required or not.